tests/performance/README.md

```diff
@@ -0,0 +1,7 @@
+# performance helm chart
+## values.yaml
+- `resources.vd.spec.type`: 
+    - `vi`: creates VMs with VirtualImage in `blockDeviceRefs`
+    - `vd`: creates VMs with corresponding `VirtualDisk`
+- `resources.vi.spec.type`:
+  - `pvc`: create vi with persistentVolumeClaim type
```

Comment:
Sorry, this readme is not helpful, can it be more informative?

- Why this directory added to repo?
- What we test and how?
- Brief info about tools/evicter and statistic.
- Example of how to start tests.